Source/Particles/Pusher/GetAndSetPosition.H

```diff
@@ -15,6 +15,29 @@
 
 #include <limits>
 
+/** \brief Extract the cartesian position coordinates of the particle
+ *         p and store them in the variables `x`, `y`, `z`
+ *         This is the implementaton and can't be called directly */
+template <typename P>
+AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+void getParticlePosition (const P& p, const RType theta,
```

Comment:
Is our convention `_` for functions?
```suggestion
void get_particle_position (const P& p, const RType theta,
```